docs/source/kedro_project_setup/configuration.md

```diff
@@ -61,16 +82,17 @@ If you both specify the `KEDRO_ENV` environment variable and provide the `--env`
 
 ## Template configuration
 
-Kedro also provides an extension [TemplatedConfigLoader](/kedro.config.TemplatedConfigLoader) class that allows you to template values in configuration files. To apply templating in your project, set the `CONFIG_LOADER_CLASS` constant in your `src/<package_name>/settings.py`:
+Kedro also provides an extension [TemplatedConfigLoader](/kedro.config.TemplatedConfigLoader) class that allows you to template values in configuration files. To apply templating in your project, set the `CONFIG_LOADER_CLASS` constant in your [`src/<package_name>/settings.py`](settings.md):
```

Comment:
Does this link work? What does it point to?

Reply:
It does work and links to: https://kedro.readthedocs.io/en/stable/kedro_project_setup/settings.html

I added the link because it's there in other parts on this page as well. It explains what `settings.py` is for, but perhaps it's not super useful here.

Reply:
Oh, OK, sorry, I was reading the link backwards for some reason 🤦‍♀️